commitizen/cz/conventional_commits/conventional_commits.py

```diff
@@ -45,55 +45,68 @@ def questions(self) -> List[Dict[str, Any]]:
                 "type": "list",
                 "name": "prefix",
                 "message": "Select the type of change you are committing",
+                "use_shortcuts": self.config.settings['use_shortcuts'],
```

Comment:
How about make it `settings['cz_conventional_commits']['use_shortcuts']`? This configuration might not make sense in other cz rules.

Reply:
Sure, conventional is the only pre-defined ruleset that uses a list select, but keep in mind that customized cz rules may also wish to use shortcuts.

Reply:
Yep, that sounds reasonable. Once the documentation is updated. I think we're good to merge it. Thanks for this awesome feature!

Reply:
I moved this check from here to a filter in the actual commit, so that it automatically covers customized cz rules. Now use_shortcuts can apply to cz_customize in config files, and to classes defining `questions()` of their own.